SlamTests/Slam/Queue/QueueViewSpec.swift

```diff
@@ -20,30 +21,84 @@ class QueueViewSpec: QuickSpec {
 
             describe("The number of rows") {
                 it("is equal to the number of matches") {
-                    queueView!.matches = [Match(matchData: ["playerOne":"One","playerTwo":"two"])]
+                    queueView!.matches = [Match(matchData: matchData)]
                     expect(queueView!.tableView(display!, numberOfRowsInSection: 0)).to(equal((1)))
                 }
             }
 
             describe("Building a table view cell") {
                 it("includes player one in the cell's label") {
-                    let match = Match(matchData: ["playerOne":"One","playerTwo":"Two"])
+                    let match = Match(matchData: matchData)
                     queueView!.showMatches([match])
                     let cell = queueView!.tableView(display!, cellForRowAtIndexPath: NSIndexPath(forRow: 0, inSection: 0))
                     expect(cell.textLabel!.text).to(contain("One"))
                 }
 
                 it("includes player two in the cell's label") {
-                    let match = Match(matchData: ["playerOne":"One","playerTwo":"Two"])
+                    let match = Match(matchData: matchData)
                     queueView!.showMatches([match])
                     let cell = queueView!.tableView(display!, cellForRowAtIndexPath: NSIndexPath(forRow: 0, inSection: 0))
                     expect(cell.textLabel!.text).to(contain("Two"))
                 }
             }
 
+            describe("removing match by slide") {
```

Comment:
The action seems to be more generic than "removing a match." What about `describe("editing a match")`?